client/scripts/controllers/server/reactions.ts

```diff
@@ -25,7 +25,7 @@ const modelFromServer = (reaction) => {
 };
 
 class ReactionsController {
-  private _store: ReactionsStore = new ReactionsStore();
+  private _store: ReactionStore = new ReactionStore();
```

Comment:
Renamed these because there was inconsistent singular/plural patterns in store. Singular seemed more prevalent, so leaned that direction.

Reply:
Good idea!